cudl-production/terraform.tfvars

```diff
@@ -51,12 +68,15 @@ db-lambda-information = [
     "description"   = "Transforms the UI json file into a json format with suitable paths for the viewer / db"
     "jar_path"      = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/0.15/AWSLambda_Data_Transform-0.15-jar-with-dependencies.jar"
     "queue_name"    = "CUDLPackageDataUIQueue"
+    "transcription" = false
     "timeout"       = 900
     "memory"        = 512
-    "filter_prefix" = "cudl.ui.json"
     "handler"       = "uk.ac.cam.lib.cudl.awslambda.handlers.UIFileDBHandler::handleRequest"
     "runtime"       = "java11"
   }
+]
+db-lambda-information = [
+
 ]
```

Comment:
Rather than passing in an empty list here, I think it would make sense to remove the variable `db-lambda-information` from the `cudl-data-processing` module as It is no longer used

Reply:
Removed.